_posts/2021-04-20-custom-errors.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: post
+published: true
+title:  Custom Errors in Solidity
+date:   2021-04-20
```

Comment:
A note to self to update the date before merging.

Reply:
And the file name